golem_core/low/exceptions.py

```diff
@@ -43,7 +41,11 @@ def __init__(self, network: str, driver: str):
 
         #   NOTE: we don't really care about this sort of compatibility, but this is
         #         a message developers are used to so maybe it's worth reusing
-        msg = str(NoPaymentAccountError(driver, network))
+        (
+            f"No payment account available for driver `{driver}`"
+            f" and network `{network}`"
+        )
+        msg = f"No payment account available for driver `{driver}` and network `{network}`"
```

Comment:
Is that variable really needed?

Reply:
It is like this in all custom exception here